### DISCUSSION TITLE
Hide Register button

### DISCUSSION THREAD

**Discussion by KumarKaruppaiah** [WeblateOrg/weblate#7323] [Q&A] [answered]
Hi,

Can anyone please help me to hide the **Register** button in Weblate? 

It is not required for us since we have integrated with Azure AD.

Thanks
Kumar

**Comment by nijel** [comment, marked as answer]
See https://docs.weblate.org/en/latest/admin/config.html#std-setting-REGISTRATION_OPEN

**Reply by KumarKaruppaiah** [reply]
Thank you so much. It is working now.
